test/parallel/test-dgram-multicast-setIf.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
```

Comment:
I like the order with regards to dgram-multicast for tab completion and relatedness, so maybe:
test/parallel/test-dgram-multicast-set-interface.js ?
(set_if is the only lowercase I would parse as an _IF ioctl and IMO mixing '-'&'_' is worse than mixed case.)